easy_timezones/middleware.py

```diff
@@ -84,7 +93,7 @@ def process_request(self, request):
         if tz:
             timezone.activate(tz)
             request.session['django_timezone'] = str(tz)
-            if getattr(settings, 'AUTH_USER_MODEL', None):
+            if getattr(settings, 'AUTH_USER_MODEL', None) and getattr(request, 'user', None):
```

Comment:
Django 1.7 fails without this (and without specifying `AUTH_USER_MODEL` in settings)...